app/javascript/components/App.js

```diff
@@ -45,6 +58,7 @@ class App extends Component {
             <Route path="/apartmentshow" component={ApartmentShow} />
             <Route path="/apartmentnew" component={ApartmentNew} />
             <Route path="/apartmentedit" component={ApartmentEdit} />
+            <Route path="/apartmentedit" render = {(props) => <ApartmentEdit {...props} apartments={this.state.apartments} />} />
```

Comment:
It looks like you have 2 paths to apartment edit.  Also, how do you know which apartment to edit?